src/components/concept-page/concept-page.styles.scss

```diff
@@ -0,0 +1,69 @@
+.cp-page-break {
+    height: 40vh;
+    width:100%;
+    color: white;
+}
+
+.cp-page-container {
+    height: 120vh;
+    width: 100vw;
+    text-align: start;
+}
+
+.cp-title {
+    font-family: 'Times New Roman', Times, serif;
+    font-size: 3em;
+    color: #5A4537;
+    letter-spacing: 4.32px;
+    display: flex;
+    justify-content: center;
+}
+
+.cp-subheading {
+    font-family: YuMincho;
+    font-size: 1.6em;
+    color: #5A4537;
+    padding-top: 13.5vh;
+    padding-left: 18vw;
+    line-height: 2;
+}
+
+.cp-two-halfs-container {
+    display: flex;  
+    padding-top: 8vh;
+}
+
+.cp-left-half-container {
+    width: 50vw;
+}
+
+.cp-textbox-container {
+    font-family: YuGothic;
+    font-size: 1em;
+    color: #5A4537;
+    letter-spacing: 0.8px;
+    padding-left: 18vw;
+    text-align: start;
+    line-height: 2;
+}
+
+.cp-textbox1 {
+    margin-bottom: 7vh;
+}
+
+.cp-button-container {
+    padding-left: 18vw;
+    padding-top: 10vh;
```

Comment:
when changing the height of the browser, the gaps in between the text boxes are changing because the unit `vh` is used.
This vertical spacing is not dependent on the view height, so you can just simply use pixels.

Reply:
This was intentional, to fill up some of the white space on a bigger screen. 
But can change it to pixels if that is preferred.

Reply:
it's not the worst thing but it's unusual and unnecessary.
the user will be scrolling vertically so there is no reason for vertical spacing to dynamically change.

Reply:
Changed to pixels